modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerCheckpointingTest.java

```diff
@@ -16,22 +16,47 @@
 
 package org.apache.ignite.util;
 
+import org.apache.ignite.configuration.IgniteConfiguration;
+import org.apache.ignite.internal.IgniteEx;
 import org.apache.ignite.internal.processors.cache.persistence.GridCacheDatabaseSharedManager;
 import org.apache.ignite.internal.processors.cache.persistence.checkpoint.CheckpointEntry;
 import org.apache.ignite.internal.processors.cache.persistence.checkpoint.CheckpointHistory;
+import org.jetbrains.annotations.NotNull;
 import org.junit.Before;
 import org.junit.Test;
 
 import static org.apache.ignite.internal.commandline.CommandHandler.EXIT_CODE_OK;
 import static org.apache.ignite.testframework.GridTestUtils.assertContains;
 
+/**
+ * Checks --checkpoint command
+ */
 public class GridCommandHandlerCheckpointingTest extends GridCommandHandlerClusterByClassAbstractTest {
     /** */
     @Before
     public void init() {
         injectTestSystemOut();
     }
 
+    /** {@inheritDoc} */
+    @Override protected void beforeTestsStarted() throws Exception {
+        assertTrue("Need at least 2 nodes to test checkpoint command", SERVER_NODE_CNT >= 2);
```

Comment:
You can indicate in a comment why this check is needed, it is not obvious to me why one node is not enough.